src/schemas/HederaUtils.ts

```diff
@@ -158,7 +158,7 @@ export function decodeSolidityErrorMessage(message: string | null): string | nul
         } else if (!message.startsWith("0x")) {
             result = message
         } else {
-            result = null;
+            result = ethers.toUtf8String(message);
```

Comment:
This is good.

Does decoding always result in legible text?
If not should we make this a flow where the user could click a button to decode on desire?
The hex might still be valuable to some devs

Reply:
Right so ethers.toUtf8String will either return an legible text or it will throw a decoding exception, and this is wrapped in a try catch, so when the program catch the exception, the `result` will be set to `null`, and when `result = null`, hashscan will show the original hex string of the error message.

Reply:
Looks good to me.